src/components/React/Howto.js

```diff
@@ -25,99 +25,105 @@ export default function Howto(props) {
           </Navbar>
         </ModalHeader>
 
-        <ModalBody>
+        <ModalBody className="px-4">
           <h2 id="user-guide" className="mb-4">
             User Guide
           </h2>
-          <Row>
+          <Row className="my-4">
             <p>
-              Welcome to the Web-VR Volume viewer. This short guide is meant to
-              help new users on to use the application and and showcase the
-              potential its potential for future research.
+              Welcome to the Narragansett Bay Volume Viewer. This short guide is
+              meant to help new users on how to navigate and analyze the 3D
+              volume data.
             </p>
             <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/guide12.png"
+              src="assets/images/howto/riddc-intro-guide.png"
```

Comment:
I don't think that this first image provides any information. Does it? I would vote to remove it

Reply:
@kmilo9999 This first one is the one that doesn't provide any information and it's not needed